crates/bevy_pbr/src/render/mesh_functions.wgsl

```diff
@@ -4,6 +4,26 @@
 #import bevy_pbr::mesh_bindings       mesh
 #import bevy_pbr::mesh_types          MESH_FLAGS_SIGN_DETERMINANT_MODEL_3X3_BIT
 
+fn affine_to_square(affine: mat3x4<f32>) -> mat4x4<f32> {
+    return transpose(mat4x4<f32>(
+        affine[0],
+        affine[1],
+        affine[2],
+        vec4<f32>(0.0, 0.0, 0.0, 1.0),
+    ));
+}
+
+fn mat2x4_f32_to_mat3x3(
```

Comment:
```suggestion
fn mat2x4_f32_to_3x3m(
```
this currently needs to end with a non-number to be usable outside the module ...

Reply:
I think I'll add an underscore. What causes this limitation in naga_oil by the way? It's usual to not allow variables/functions to begin with a digit, but ending is usually fine.

Reply:
sorry this still doesn't work. the problem is that items exported from modules need to keep the same name after naga wrangling, and items that end with numbers are used by naga for duplicated function names (which can't occur normally, but are allowed internally). so they get modified to disambiguate and hence are not exported as the name you give them.

underscores are not allowed because it's invalid in glsl.

both these restrictions will be removed when [nana_oil#41](https://github.com/bevyengine/naga_oil/pull/41) lands,